### PR TITLE
Create hello.txt

### PR DESCRIPTION
**Pull request by tech101-chastina**
I would like to add a new file.

/cc @ChastinaLi

### CODE REVIEW

File: hello.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+hello
```
Comment (ChastinaLi) [inline hello.txt]: Should it be capitalized?

**Review by ChastinaLi — CHANGES_REQUESTED** [review]
plz see my comment.

**Comment by tech101-chastina** [comment]
@ChastinaLi plz see changes

**Comment by ChastinaLi** [comment]
Looks good! 🚢 it!

**Review by ChastinaLi — APPROVED** [review]

**Pull request merged**